Automattic/wp-calypso
# Gutenframe: Fetch the frame nonce directly from the v1.3 endpoint.

#### Changes proposed in this Pull Request

* We need to keep `requestSites` on v1.2 because of other dependencies (we also have a long tail of Jetpack sites that remain on <v7.3 for whatever reason). Since Gutenframe is the only feature requiring the updated Jetpack-compatible frame nonces, have the component fetch it directly from v1.3 for the current site.

#### Testing instructions

* Load a test Jetpack site (<v7.3) at `/block-editor/post/:site` and verify you are sent to Calypsoifed `wp-admin`.
* Update the Jetpack site to >=7.3, and at the same path, verify you now successfully load the iframed editor.
